src/Resolvers/Microsoft.DotNet.NativeWrapper/Interop.cs

```diff
@@ -57,7 +57,7 @@ internal enum hostfxr_resolve_sdk2_result_key_t : int
         [StructLayout(LayoutKind.Sequential, CharSet = CharSet.Auto)]
         internal struct hostfxr_dotnet_environment_info
         {
-            public int size;
+            public nuint size;
             public string hostfxr_version;
             public string hostfxr_commit_hash;
             public int sdk_count;
```

Comment:
`sdk_count` and `framework_count` should also be `nuint`

Reply:
Sounds good, I've made that update.